app/assets/stylesheets/bubble.scss

```diff
@@ -0,0 +1,50 @@
+$white: #ffffff;
+$oil: #333333;
+
+.bubble {
+  position: relative;
```

Comment:
Properties should be ordered background, border, -moz-border-radius, -webkit-border-radius, border-radius, -moz-box-shadow, -webkit-box-shadow, box-shadow, margin, min-height, padding, position, width